graphql/batch_executor.go

```diff
@@ -101,7 +101,7 @@ func (e *BatchExecutor) Execute(ctx context.Context, typ Type, source interface{
 	if topLevelRespWriter.errRecorder.err != nil {
 		return nil, topLevelRespWriter.errRecorder.err
 	}
-	return writers, nil
+	return outputNodeToJSON(writers), nil
```

Comment:
Can tests now stop calling this?

Reply:
don't know what you mean...

Reply:
for some reason I thought there were tests using this outputNodeToJSON before diffing which now might not be necessary anymore, but I can't find any tests doing that.